src/index.test.js

```diff
@@ -15,6 +15,7 @@ describe('index', () => {
     Object.keys(spies).map(x => spies[x].mockClear());
   });
 
+  jest.setTimeout(15000);
```

Comment:
put it inside the relevant test (and not on the class level)

Reply:
I will move this, but Its doesn't matter in fact